### PR TITLE
fix: ci for mac

### PR DESCRIPTION
**Pull request by eldu**
https://www.electronforge.io/guides/code-signing/code-signing-macos#osxsign-options

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
I'm going to merge this back in, thanks so much @eldu!

File: forge.config.js [hunk 1]
```diff
@@ -29,6 +30,7 @@ export default {
       config: {
         options: {
           icon: "assets/icons/icon.png",
+          format: "ULFO",
```
Comment (eldu) [inline forge.config.js:33]:
```suggestion
```

Comment (eldu) [inline forge.config.js]: @RobertGemmaJr we can delete this line! I added this in but it doesn't seem to do anything

Comment (eldu) [reply forge.config.js]:
```suggestion
```

**Pull request merged**